all_test.go

```diff
@@ -1430,8 +1430,8 @@ func TestAuthArchiveIdentity(t *testing.T) {
 	// Using an ldap backend, authentication will succeed, as we cannot archive users on the ldap system.
 	if !*backend_ldap {
 		// Try to authenticate with archived user
-		if _, err := c.GetTokenFromIdentityPassword(joeEmail, joePwd); err != ErrIdentityAuthNotFound {
-			t.Fatalf("TestArchiveIdentity failed, archived user should not be allowed to authenticate: %v", err)
+		if _, err := c.GetTokenFromIdentityPassword(joeEmail, joePwd); err == nil {
+			t.Fatalf("Archived user should not be allowed to authenticate: %v", err)
```

Comment:
is the %v needed since its nil? same for the rest below

Reply:
I just thought I was being a good Samaritan, cleaning up Stewart's old cruft.. but sure.. you're not wrong!